sources/Context.cpp

```diff
@@ -199,6 +200,29 @@ void Context::filterPartiallyConnectedDynamicModels() {
   }
 }
 
+void Context::removeRpclFromGenerators(const std::vector<inputs::AssemblingDataBase::MacroConnect>& automatonGeneratorMacroConnects) {
+  for (const auto &automatonGeneratorMacroConnect : automatonGeneratorMacroConnects) {
```

Comment:
I wonder if it wouldn't be better to do this in the Dynamic models/Generator algorithms

Reply:
Before we call Context::walkNodesMain(), dynamicModels_.models and generators_ are empty

That's why I thought it was better to call removeRpclFromGenerators in filterPartiallyConnectedDynamicModels()

Reply:
yes but it is not the role of context to define models, we need to think on how to do this properly